api/openapi.yml

```diff
@@ -540,6 +540,46 @@ paths:
           description: Unauthorized
```

Comment:
This file is missing the change to the handlers.UserInfo schema that was made.

Reply:
Good spot. Will fix.